oaiharvest/metadata.py

```diff
@@ -45,9 +45,10 @@ def writeMetadata(self, metadata_prefix, element, metadata):
 class XMLMetadataReader(object):
     """Really simple MetadataReader to serialize metadata to pretty XML."""
     def __call__(self, metadata_element):
-        return '\n'.join([tostring(rec_element,
-                                   method="xml",
-                                   pretty_print=True
-                                   )
-                          for rec_element
-                          in metadata_element])
+        # Six call fixes 'sequence item 0: expected str instance, bytes found'
+        return ('\n' if six.PY2 else b'\n').join(
```

Comment:
Which version of Python 3 are you testing with? According to the [lxml docs](http://lxml.de/FAQ.html#general-questions) `tostring` should return unicode in Python >=3.3, which would be compatible with the simple `\n`.